templates/deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: metadata-service
+  labels:
+    k8s-app: metadata-service
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: metadata-service
+  template:
+    metadata:
+      annotations:
+        fluentd.packet.net/efk: stdout+stderr
+        fluentd.packet.net/efk-index: metadata-service
```

Comment:
This shouldn't be needed? And if they are, since this chart is open source this should be set in the values file as an optional param.

Reply:
Ok, removed.